### PR TITLE
Feat/csp

### PR DESCRIPTION
**Pull request by ajuvercr**
After #40

### CODE REVIEW

File: lib/condition/condition.ts [hunk 1]
```diff
@@ -28,13 +28,13 @@ export function empty_condition(): Condition {
 export function parse_condition(source: string, baseIRI: string): Condition {
     const shapeQuads = new Parser().parse(SHAPES);
     const output = extractShapes(shapeQuads, {
-        "https://w3id.org/tree#And": (obj) =>
+        "http://vocab.deri.ie/csp#And": (obj) =>
```
Comment (smessie) [inline lib/condition/condition.ts:31]: Wanted to check out the vocabulary, but cannot dereference it :(

File: lib/condition/condition.ts [hunk 2]
```diff
@@ -551,31 +551,27 @@ export class LeafCondition implements Condition {
...
+    items: Condition[];
 
     private logger = getLoggerFor(this);
 
-    constructor(inp: { alpha: Condition; beta: Condition }) {
-        this.alpha = inp.alpha;
-        this.beta = inp.beta;
+    constructor(inp: { items: Condition[] }) {
+        this.items = inp.items;
+        console.log(inp.items);
```
Comment (smessie) [inline lib/condition/condition.ts:560]: Still a console.log left behind

File: lib/condition/condition.ts (hunk 1 above)
Comment (ajuvercr) [reply lib/condition/condition.ts:31]: it's this boy https://lov.linkeddata.es/dataset/lov/vocabs/csp

Comment (smessie) [reply lib/condition/condition.ts:31]: Yeah, I also found https://lov.linkeddata.es/dataset/lov/terms/?&type=class&tag=Industry&vocab=csp listing the different URIs, but still sad that it doesn't dereference.

**Review by smessie — APPROVED** [review]

**Pull request merged**